homeassistant/components/media_player/kef.py

```diff
@@ -0,0 +1,302 @@
+"""
+Support for interfacing HA with the KEF Wireless Speakers .
+
+For more details about this platform, please refer to the documentation at
+https://github.com/Gronis/pykef
```

Comment:
The URL must point to our documentation.

Reply:
Do you suggest that I change it to the docs in the PR for home-assistant.io?

Reply:
Ok, then it is resolved.